Centralized Discovery Controller (CDC)/SFSS/docker-compose.yml

```diff
@@ -6,87 +6,87 @@ version: "3.7"
 services:
 
   alertservice:
-    image: "ghcr.io/dell/SANdbox/alertservice:1.1.0.0038"
```

Comment:
It looks like this change is causing the CI check to fail?

Reply:
this is new CI that wasn't running before, fixed for now. Will update the full path again once we publish dockers...